solutions/src/test/java/ru/sberbank/school/task11/StreamsTest.java

```diff
@@ -0,0 +1,81 @@
+package ru.sberbank.school.task11;
+
+import lombok.Setter;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+public class StreamsTest {
+
+    private Person[] array;
+    private List<Person> collection;
+
+
+    @Before
+    public void initialize() {
+        array = new Person[9];
+        collection = new ArrayList<>();
+        collection.add(new Person("Jhon", 23));//0
+        collection.add(new Person("Jack", 24));//1
+        collection.add(new Person("Sam", 18));//2
+        collection.add(new Person("James", 25));//3
+        collection.add(new Person("Poly", 38));//4
+        collection.add(new Person("Mike", 30));//5
+        collection.add(new Person("Enjoikin", 33));//6
+        collection.add(new Person("Michale", 31));//7
+        collection.add(new Person("Tom", 41));//8
+        array = collection.toArray(array);
+    }
+
+    /**
+     * of(Collection<T> elements) + filter + sorted + ToMap
```

Comment:
Javadoc comment at column 21 has parse error. Missed HTML close tag &#39;T&#39;. Sometimes it means that close tag missed for one of previous tags.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1417249004>*